src/pages/transactions/tx.tsx

```diff
@@ -62,7 +62,9 @@ const SingleTransaction: NextPage = () => {
       <Breadcrumbs Icon={TransactionsIcon} first="Transactions" second="Details" firstLink={breadcrumbsLink} />
 
       {(error || !isCurrentSafeTx) && !loading ? (
-        <ErrorMessage error={error}>Failed to load transaction {transactionId}</ErrorMessage>
+        <ErrorMessage error={!isCurrentSafeTx ? new Error('Error: tx Id not found in this Safe') : error}>
```

Comment:
Not totally sure if we should be overwriting the `error` here but we were getting
> undefined: undefined